src/main/kotlin/org/malachite/estella/commons/OwnResponses.kt

```diff
@@ -0,0 +1,7 @@
+package org.malachite.estella.commons
+
+import org.springframework.http.ResponseEntity
+
+object OwnResponses{
+    val UNAUTH: ResponseEntity<String> = ResponseEntity.status(401).body("Unauthenticated")
```

Comment:
Zrób tu zamiast 
```kotlin
ResponseEntity.status(401).body("Unauthenticated")
```
to:
```kotlin
ResponseEntity.status(401).body(Message("Unanthenticated")
```
Już wysyłajmy te wiadomości, będzie lepiej na froncie z tego korzystać

Reply:
Plus od razu możesz dorzucić tutaj SUCCESS, jak już coś takiego mamy

Reply:
Znaczy to miałem zamiar zrobić jak zmerguje od ciebie wiadomości więc zaraz się to zrobi

Reply:
Okok